### DISCUSSION TITLE
Offset boolean

### DISCUSSION THREAD

**Discussion by ralphgeorg-s** [libigl/libigl#2036] [Q&A]
if using a boolen difference - I need an offset - which means the space between the objects should be for example 50 microns. let's say I have a cube and a pipe - I create a hole in the cube exactly the same size as the pipe. adding an offset - the hole would be bigger than the pipe like 50 microns. that's a simple example but the objects we use are more complex, so we cannot just increase the size - we need to do it with an offset. 

any advice or ideas? thanks.

**Comment by alecjacobson** [comment]
In the past the way that I do this is to first compute a signed distance
offset_surface as a mesh then use that in the boolean
